src/libcore/macros.rs

```diff
@@ -337,21 +337,16 @@ macro_rules! try {
 
 /// Write formatted data into a buffer
 ///
-/// This macro accepts a 'writer' (any value with a `write_fmt` method), a format string, and a
-/// list of arguments to format.
+/// This macro accepts a format string, a list of arguments, and a 'writer'.
+/// Arguments will be formatted according to the specified format string and the result will be
```

Comment:
It's all good! I tend to over-whitespace. If you want it in one paragraph, can you then remove the `\n`s so that's more obvious? I hear you about diffs, but we haven't found it too onerous, and so don't worry about that.